src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -77,12 +79,21 @@ public BusinessesService(List<Listing> allBusinesses) {
             .apiKey(KEY)
             .build();
     TextSearchRequest request = PlacesApi.textSearchQuery(context, product);
-    
+    int radius = 1500;
```

Comment:
Please make 1500 a constant. Sorry I missed this in my last review. Also, why is this radius different from the already named radius above?

Reply:
1500 is not a constant, it is the initial radius for the text search request and if enough results are not returned, I increase it by the RADIUS_MULTIPLIER. However, the MAX_ALLOWED_TEXT_SEARCH_RADIUS variable is the max number that the radius can be for the text search request. I make sure the current radius is not greater then this just in case someone puts in a zip code that naturally has not that many results will allow me not to make to many requests

Reply:
RIght. This is currently an integer literal. Given that it has some meaning to the code I am suggesting it be converted to a defined constant to provide that context. In both cases the value will not change. Here is a reference: https://www.diffen.com/difference/Constant_vs_Literal

In regards to the varying radii, I see. Thank you for clarifying. Understanding this is a starting radius, it can still be converted to a constant :)

Reply:
Okay I think I was confused what you meant by make it a constant variable. However, I see now that you wanted me to make the number 1500 a constant and use it within the function.